src/server/system_services/schemas/account_schema.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
         next_password_change: { date: true },
 
         // default policy for new buckets
-        default_pool: { objectid: true },
+        default_resource: { objectid: true },
```

Comment:
This requires conversion of existing accounts... but i didn't see it included in the pr..

Reply:
correct thanks, added upgrade for accounts and namespace buckets.